workspaces/ui-v2/src/pages/diffs/components/DiffAccessoryNavigation.tsx

```diff
@@ -18,11 +18,16 @@ export function DiffAccessoryNavigation() {
     handledCount,
     hasDiffChanges,
     getUndocumentedUrls,
+    wipPatterns,
   } = useSharedDiffContext();
   const diffReviewCapturePage = useDiffReviewCapturePageLink();
   const undocumentedUrlsPageLink = useDiffUndocumentedUrlsPageLink();
   const history = useHistory();
   const [handled, total] = handledCount;
+  const hasChanges =
+    hasDiffChanges() ||
+    Object.values(wipPatterns).filter((pattern) => pattern.isParameterized)
+      .length > 0;
```

Comment:
Total nit: this naming confuses me, do the `wipPatterns` have anything to do with whether there are any changes? Seems more like `shouldAskForCommit = hasDiffChanges() || isDiffsPage`?

Reply:
wipPatterns refers to the parameterized path patterns here 
<img width="634" alt="Screen Shot 2021-08-11 at 9 57 21 AM" src="https://user-images.githubusercontent.com/18374483/129042224-af36bf52-b6f0-4ec9-a686-f8fd268df4e2.png">

There is user input, but nothing to "save" until you've learnt a path. We've seen users navigate away while this is in progress and then come back and lose their data - therefore we should have a prompt (partially completed form)